app/models/efile_submission_transition.rb

```diff
@@ -31,16 +31,26 @@ def initiated_by
     User.find(metadata["initiated_by_id"])
   end
 
-  def error_message
-    metadata["error_message"]
-  end
+  def stored_errors
```

Comment:
`errors` as a method name is reserved by ActiveModel which is a bummer because it would have been the perfect name for a method here.